po/pt_BR.po

```diff
@@ -1,18 +1,18 @@
 # Portuguese translations for com.github.sgpthomas.hourglass package.
 # Copyright (C) 2020 THE com.github.sgpthomas.hourglass'S COPYRIGHT HOLDER
 # This file is distributed under the same license as the com.github.sgpthomas.hourglass package.
-# Cleiton Floss <cleitonfloss@gmail.com>, 2020.
+# Cleiton Floss <gfavaro@gmail.com>, 2020.
```

Comment:
```suggestion
# Cleiton Floss <cleitonfloss@gmail.com>, 2020.
# Gabriel Fávaro <gfavaro@gmail.com>, 2020.
```

You might want to keep the previous translator as it is and just add your name and email here in a new line.